src/mixins/common-mixin.ts

```diff
@@ -179,33 +173,39 @@ export class CommonMixin extends Vue {
     ].includes(nr?.legalType)
   }
 
+  /** Returns true if the specified entity type is allowed for incorporation / registration. */
+  isSupportedIncorporationRegistration (type: EntityTypes): boolean {
+    const supportedEntites = GetFeatureFlag('supported-incorporation-registration-entities') || []
+    return supportedEntites.includes(type)
+  }
+
   /** Returns true if the specified request type is allowed for alteration (conversion). */
   isSupportedAlteration (type: NrRequestTypeCodes): boolean {
-    const supportedAlterationTypes = GetFeatureFlag('supported-alteration-types')
+    const supportedAlterationTypes = GetFeatureFlag('supported-alteration-types') || []
```

Comment:
What do you think of adding these to the `defaultFlagSet` Sev?

Reply:
I was being lazy, but yes, that's ideal. Changing it now...

Reply:
Done.

Reply:
Fixed, thanks Sev! Resolving.